### ISSUE TITLE
ImportError: cannot import name SSL2HandshakeType

### ISSUE THREAD

**Issue by arainho** [tomato42/tlsfuzzer#31]
Hello

I got this issue when running test-sslv2-force-export-cipher.py.
> ImportError: cannot import name SSL2HandshakeType

I have the recommended  tlslite-ng version
```
$ pip2 list | grep tlslite-ng
tlslite-ng (0.6.0a4)
```

I run the python script with:
```
$ PYTHONPATH=. python2 scripts/test-sslv2-force-export-cipher.py -h example.com -p 443

Traceback (most recent call last):
  File "scripts/test-sslv2-force-export-cipher.py", line 9, in <module>
    from tlsfuzzer.runner import Runner
  File "/home/Code/drown_attack/tlsfuzzer/tlsfuzzer/runner.py", line 11, in <module>
    from tlslite.constants import ContentType, HandshakeType, AlertLevel, \
ImportError: cannot import name SSL2HandshakeType
```

**Comment by DukeHarris** [comment]
Same here 
tlslite-ng (0.6.0a4)
Python 3.5.1

**Comment by tomato42** [comment]
that's because the ssl2 branch of tlsfuzzer requires sslv2 branch of tlslite-ng

as soon as the sslv2 branch of tlslite-ng is released, I will merge the ssl2 branch of tlsfuzzer

**Comment by tomato42** [comment]
should be solved with `tlslite-ng 0.6.0a5`

**Comment by tomato42** [comment]
The scripts are under CI and are confirmed to run under all python versions so I'm closing this issue.
